src/converse-chatview.js

```diff
@@ -67,6 +67,8 @@ converse.plugins.add('converse-chatview', {
                 'emoji': true,
                 'spoiler': true
             },
+            'image_urls_regex': null,
+            'image_domains_whitelist': null
```

Comment:
Instead of adding a new configuration setting, we can re-use [show_images_inline](https://github.com/conversejs/converse.js/blob/823134d7629bad2b82875964169340609c0ce2c9/docs/source/configuration.rst#L1681).

If `show_images_inline` is set to `true`, then all domains are whitelisted. If an array is passed in, then only those domains are whitelisted.

Please remember to update the configuration documentation: https://github.com/conversejs/converse.js/blob/master/docs/source/configuration.rst

And please also add a test that checks image domain whitelisting.